### ISSUE TITLE
Adding Reloader deployment to osc-cl2 cluster

### ISSUE THREAD

**Issue by dystewart** [operate-first/apps#1778]
We need to add a Reloader deployment to osc-cl2 cluster to reload the trino coordinator pod.

**Comment by dystewart** [comment]
I'll be opening a PR for this shortly

**Comment by dystewart** [comment]
Once #1782 is merged I will create a new PR to add annotations to the trino coordinator deployment in [https://github.com/operate-first/odh-manifests/tree/osc-cl2-v1.1.2](https://github.com/operate-first/odh-manifests/tree/osc-cl2-v1.1.2)
